app/assets/stylesheets/govuk_publishing_components/components/_metadata.scss

```diff
@@ -23,16 +23,20 @@
 }
 
 .gem-c-metadata--inverse {
+  background-color: govuk-colour("blue");
   color: govuk-colour("white");
+  padding: govuk-spacing(2);
 
-  a:link,
-  a:active,
-  a:visited {
-    color: govuk-colour("white");
+  a {
+    @include govuk-link-style-inverse;
   }
 
-  a:focus {
-    color: govuk-colour("black");
+  .gem-c-metadata__list {
```

Comment:
I'm wondering if we should move `.gem-c-metadata__list` out of `.gem-c-metadata--inverse`, allowing it to be used in other contexts in the future.

Not sure how likely this is, what do you think?

Reply:
Good point - I was thinking about that but, at least for now, we only need the spacing changes with inverse enabled since it allows for consistent spacing. I think if I try to move it outside of `gem-c-metadata--inverse` I might end up causing spacing issues (without the inverse option).

Reply:
Thanks, make sense, I'll approve the changes now 👍